CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -315,6 +315,8 @@ public function upgrade_4_7_16($rev) {
   public function upgrade_4_7_18($rev) {
     $this->addTask('Update Kenyan Provinces', 'updateKenyanProvinces');
     $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
+    $this->addTask('Add is_public column to civicrm_custom_group', 'addColumn',
+      'civicrm_custom_group', 'is_public', "boolean DEFAULT '1' COMMENT 'Is this property public?'");
```

Comment:
@jitendrapurohit could you move the upgrade changes to ```upgrade_4_7_19(...)``` [here](https://github.com/civicrm/civicrm-core/blob/master/CRM/Upgrade/Incremental/php/FourSeven.php#L332) ?